boskos/cleaner/cmd/main.go

```diff
@@ -41,11 +41,15 @@ var (
 	kubeClientOptions crds.KubernetesClientOptions
 
 	boskosURL    string
+	username     string
+	password     string
```

Comment:
nit: this is not a password, but a password file -- let's update the name

Reply:
Done.